common.py

```diff
@@ -69,9 +69,10 @@ def __init__(self, dictionary):
     def write(self):
         """This function writes the data to elastic search"""
         self.dictionary = json.dumps(self.dictionary)
-        requests.post('http://elasticsearch2.gridpp.rl.ac.uk:9200/logstash' +
-                       '-gridtools-metrics-'
-                       + date + '/apel/', data=self.dictionary)
+        requests.post("http://elasticsearch2.gridpp.rl.ac.uk:9200/"
+                      "logstash-gridtools-metrics-%s/"
+                      "metric_data/" % date,
```

Comment:
I didn't know Python magically joined adjacent strings! This looked like an issue until I tried it myself.

Reply:
Yeah, so long as they're within some brackets. I wasn't completely sure if it would work within an argument though.